pre_commit_hooks/pretty_format_json.py

```diff
@@ -115,6 +131,9 @@ def main(argv=None):  # type: (Optional[Sequence[str]]) -> int
             if contents != pretty_contents:
                 print('File {} is not pretty-formatted'.format(json_file))
 
+                if args.show_expected:
+                    print(get_diff(contents, list(pretty_contents)))
```

Comment:
I don't think we need a new option, can just do this in the `else` condition of `if args.autofix:` probably

Reply:
Ah yeah, I didn’t see your earlier comments before this change - I’ll switch it out and make the code changes above when I get a chance!

Reply:
How does the test function look?

Reply:
A couple of things:

- you probably don't need to test how `difflib` works
- for table tests in pytest you'd use `pytest.mark.parametrize` (this will make them separate tests)
- I would just do a single test of `main` to demonstrate the behaviour